<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STT2NG</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tableTab">
       <attribute name="title">
        <string>Table View</string>
       </attribute>
       <layout class="QVBoxLayout" name="tableTabLayout">
        <item>
         <widget class="QGroupBox" name="generatorControlsGroup">
          <property name="title">
           <string>Generator Controls</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="controlsBox">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Spacing</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Node Scale</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="nodeScaleSpinBox">
                <property name="prefix">
                 <string/>
                </property>
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
                </property>
                <property name="value">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="nodeSpacingSpinBox">
                <property name="prefix">
                 <string/>
                </property>
                <property name="suffix">
                 <string/>
                </property>
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
                </property>
                <property name="value">
                 <double>30.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignBottom">
            <widget class="QPushButton" name="generateNodesButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Generate Graph</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignBottom">
            <widget class="QPushButton" name="generate2DButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Generate 2D Projection</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignBottom">
            <widget class="QPushButton" name="generate3DModelButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Generate 3D Model</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graphTab">
       <attribute name="title">
        <string>Graph View</string>
       </attribute>
       <layout class="QVBoxLayout" name="graphTabLayout">
        <item>
         <widget class="QGroupBox" name="buttonBox">
          <property name="title">
           <string>Graph Controls</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="buttonBoxLayout">
           <item alignment="Qt::AlignVCenter">
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buildButton">
             <property name="text">
              <string>Build Relation</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QDoubleSpinBox" name="toleranceSpinBox">
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::DefaultStepType</enum>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignVCenter">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Tolerance</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QWidget" name="relationOrderContainer" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item alignment="Qt::AlignVCenter">
               <widget class="QSpinBox" name="relationOrderSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignVCenter">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Relation Order</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="twoDTab">
       <attribute name="title">
        <string>2D View</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="threeDTab">
       <attribute name="title">
        <string>3D View</string>
       </attribute>
       <layout class="QHBoxLayout" name="threeDTabLayout">
        <item>
         <widget class="QGroupBox" name="threeDSettings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Settings</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="0" column="0" alignment="Qt::AlignTop">
            <widget class="QDoubleSpinBox" name="distanceScaleSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="showGroupSeparator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignTop">
            <widget class="QPushButton" name="setDistanceScaleButton">
             <property name="text">
              <string>Set Distance Scale</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Table</string>
    </property>
    <addaction name="actionImport_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Parameters"/>
    <addaction name="actionLoad_Parameters"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QMenu" name="menuExport_as">
     <property name="title">
      <string>Export as...</string>
     </property>
     <addaction name="actionExport_CSV"/>
    </widget>
    <addaction name="actionImport_Event"/>
    <addaction name="separator"/>
    <addaction name="menuExport_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_screenshot"/>
   </widget>
   <addaction name="menuTable"/>
   <addaction name="menuGraph"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_CSV">
   <property name="text">
    <string>Import CSV</string>
   </property>
  </action>
  <action name="actionSave_Parameters">
   <property name="text">
    <string>Save Description</string>
   </property>
  </action>
  <action name="actionLoad_Parameters">
   <property name="text">
    <string>Load Description</string>
   </property>
  </action>
  <action name="actionExport_Text">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionExport_CSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionSave_screenshot">
   <property name="text">
    <string>Save screenshot</string>
   </property>
  </action>
  <action name="actionImport_Event">
   <property name="text">
    <string>Import Event(s)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
